### ISSUE TITLE
[MENTEE] Julius Schmid

### ISSUE THREAD

**Issue by jusssch** [sktime/mentoring#46]
**Mentee:** 

Julius Schmid / high school student in Germany / very interested in projects that solve problems through ml 

gh: jusssch
t: jussch_
linkedin: https://www.linkedin.com/in/jussch/ 

**Mentor(s):** 
fkiraly

**Why did you join sktime's mentorship program?**
Until now I have learned anything I wanted to know about ML through books, articles, ... on my own. I have reached a point at which I feel like I have solid understanding of the backgrounds (Neural Networks; gradient descent; backpropagation; the math behind it) but don't have practical experience with the things I learned. When I tried projects, it often felt frustrating because I don't understand a lot of the ML frameworks. That is why I think the sktime mentorship program can help me. Because the mentors have real-world experience, they can help me when I get stuck and teach me how to go about solving these issues.

**What topics are you working on?** 

Mainly my fields are mobile (iOS and partly Android) and web development. I also learned the fundamentals of interpreters and computer science.
Projects can be found on my site: juliusschmid.net

**What are your learning goals?**
- Learn everything i need so i can further learn ml concepts (sounds ironic) and get started with my own projects.
      - math + code

**What's next for you after the mentorship program?**
Working on projects that solve real-world problems through ML. Beginning small and trying to make an impact.
